markdown/chapters/appendices/playables.md

```diff
@@ -0,0 +1,278 @@
+Wishclaw or Solve the Equation would be the most viable alternatives.
+
+:card[Profane Tutor] has been experimented with a lot and can be a decent choice
+for lists. The card is best utilised on turn 2 which means, in theory, you want
+a high number of copies where possible to ensure it is found early on.
+Unfortunately, as additional later copies of the card beyond this turn are not
+useful, running high numbers of it can lead to a lot of dead draws from the
+deck. It also suffers the same issues as some of the other alternatives with
+speed being lost from its use.
+
+## Wishes
+
+:::row{variant=centered}
+- Burning Wish
+- Death Wish
+- Wish
+:::
+
+Wishes are often discussed as a way to increase the number of pseudo copies of
+Doomsday within your deck. The most common one discussed is :card[BW] in regards
+to its mana efficiency and ability to also find a number of strong *toolbox*
+spells. However because of the requirement to add in a significant red splash to
+the main deck configuration (given the majority of lists are purely blue-black
+without sideboarding) many people look to :card[Death Wish] as a possible
+alternative. Unfortunately none of the current *wish* effects are as efficient
+as the *tutor* equivalents :card[PT] and :card[LDV] in terms of mana cost and
+flexibility.
+
+They also generally promote a slower style of play which would tend to omit
+tutor effects anyway for potentially additional card advantage engines. If you
+were to run them, then they would probably replace any tutor effects in your
+deck but are not recommended.
+
+## Baleful Strix
+
+:::row{variant=centered}
+- Baleful Strix
+- Ice-Fang Coatl
+:::
+
+Baleful Strix is a great card that can sometimes feel a bit off in a combo
+shell. It acts as a multifaceted tool that both helps delay opposing aggression
+whilst still cantripping and allowing you to keep a high blue count in your card
+pool. When Strix is played it normally replaces some number of cyclers or other
+interaction slots because it tends to promote a slower pace of gameplay. Quite
+often it is run alongside other tools that synergise with it. Things like
+:card[Cabal Therapy], :card[Unearth] or a generally more controlling or
+creature-centric build.
+
+## Unearth
+
+:::row{variant=centered}
+- Unearth
+:::
+
+Unearth was very popular as the current style of the deck was explored. It
+paired well with :card[Predict] which many were running and held its own in a
+vacuum thanks to the cycling allowing it to be thrown back if unneeded. These
+days it is considered a little too cute as it requires the use of the graveyard
+and doesn't fit properly with the style of deck many play. Despite this it's
+still perfectly playable alongside copies of Strix or other creature plans and
+can function as an alternative to a second Oracle in piles.
+
+## Creatures
+
+:::row{variant=centered}
+- Uro, Titan of Nature's Wrath
+- Monastery Mentor
+- Sedgemoor Witch
+- Gurmag Angler
```

Comment:
i would add Murktide, because it is one of hte main threats of the later mentioned Tempo Doomsday